### DISCUSSION TITLE
blog/kubecon-china-2024-recap/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#355] [Announcements]
# blog/kubecon-china-2024-recap/

深入剖析 KubeCon China 2024 中的云原生技术进展，聚焦 Istio 的 Ambient 模式和 API 网关的融合及其对行业的深远影响。

https://jimmysong.io/blog/kubecon-china-2024-recap/

**Comment by scriptkids** [comment]
请问下，blog 有 rss 可订阅么？

**Reply by rootsongjc** [reply]
有的，这是订阅链接：https://jimmysong.io/blog/index.xml

**Reply by scriptkids** [reply]
thx～
